Minio/ApiEndpoints/ObjectOperations.cs

```diff
@@ -523,10 +524,10 @@ private async Task<string> PutObjectAsync(string bucketName, string objectName,
         {
             var queries = new List<string>();
             queries.Add("uploads");
-            if (prefix != null)
-            {
-                queries.Add("prefix=" + Uri.EscapeDataString(prefix));
-            }
+            queries.Add("prefix=" + Uri.EscapeDataString(prefix));
```

Comment:
Is the `prefix != null` check not required here?

Reply:
isn't it already done? if prefix is null then we set prefix = ""

Reply:
My bad fixed..